pkg/kubeapiserver/options/serving.go

```diff
@@ -39,34 +37,3 @@ func NewSecureServingOptions() *genericoptions.SecureServingOptionsWithLoopback
 	}
 	return o.WithLoopback()
 }
-
-// NewInsecureServingOptions gives default values for the kube-apiserver.
-// TODO: switch insecure serving off by default
-func NewInsecureServingOptions() *genericoptions.DeprecatedInsecureServingOptionsWithLoopback {
-	o := genericoptions.DeprecatedInsecureServingOptions{
-		BindAddress: net.ParseIP("127.0.0.1"),
-		BindPort:    8080,
-	}
-	return o.WithLoopback()
-}
-
-// DefaultAdvertiseAddress sets the field AdvertiseAddress if
-// unset. The field will be set based on the SecureServingOptions. If
-// the SecureServingOptions is not present, DefaultExternalAddress
-// will fall back to the insecure ServingOptions.
-func DefaultAdvertiseAddress(s *genericoptions.ServerRunOptions, insecure *genericoptions.DeprecatedInsecureServingOptions) error {
```

Comment:
why was this function dropped? isn't it used for secure setup as well?

Reply:
it was used to set default advertise address for DeprecatedInsecureServingOptions which we don't have anymore?

Reply:
ah, got that confused with https://github.com/kubernetes/kubernetes/blob/release-1.19/cmd/kube-apiserver/app/server.go#L595